.travis.yml

```diff
@@ -66,7 +66,7 @@ before_install:
         fi;
       fi
     # Always install these via pip so we get the latest possible versions (testing bugfixes)
-    - pip install --upgrade pytest!=5.4.0 pytest-sugar pytest-cov pytest-mock pytest-timeout codecov
+    - pip install --upgrade pytest<5.4.0 pytest-sugar pytest-cov pytest-mock pytest-timeout codecov
```

Comment:
I think you can't use `<` in a shell command, so you need to put this package in quotes. I've already done that in #7446.